publisher-sdk/src/androidTest/java/com/criteo/publisher/UserPrivacyFunctionalTest.java

```diff
@@ -129,6 +131,36 @@ public void whenCriteoInit_GivenUspOptoutTrue_ThenChangedToFalseAfterFirstCall_V
     assertEquals("false", cdb.getUser().getUspOptout());
   }
 
+  @Test
+  public void whenCriteoInit_GivenTagForChildDirectedTreatmentIsNull_VerifyItNullRegsIsPassedToCdb() throws Exception {
```

Comment:
I would not say that the tag is null. It is rather not provided, so as a default value, given no tag, then null is passed to CDB